roles/migrationcontroller/files/migration.openshift.io_directvolumemigrationprogresses.yaml

```diff
@@ -134,8 +134,6 @@ spec:
                   type: string
               type: object
             podSelector:
-              additionalProperties:
```

Comment:
The same change needs to be here? https://github.com/konveyor/mig-operator/blob/2cd6bf41c3476b70f600ec5499b069d926b34cc0/deploy/olm-catalog/bundle/manifests/migration.openshift.io_directvolumemigrationprogresses.yaml#L136